### PR TITLE
Spring 2018 Fanart Contest newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**
Contest listing not yet live, but prepared and ready.

### CODE REVIEW

File: news/2018-04-03-spring-2018-fanart-contest.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Spring 2018 Fanart Contest"
+date: 2018-04-03 07:00:00 +0000
+---
+
+We're ready to appreciate the blooming talents of our amazing artists once again, in 2018's first Fan Art contest, featuring spring! Like to draw and want some free osu!supporter? Check this out!
```
Comment (MegaApplePi) [inline news/2018-04-03-spring-2018-fanart-contest.md]: `Fan Art` -> `Fanart`?
the title and this occurrence don't match, I guess... then again, shouldn't it be `fanart contest` or `Fanart Contest`?

Comment (Ephemeralis) [reply news/2018-04-03-spring-2018-fanart-contest.md]: good pick there actually, hmmm. old articles use fanart so I guess i'll go with that to keep consistent with how things historically have been

File: news/2018-04-03-spring-2018-fanart-contest.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+date: 2018-04-03 07:00:00 +0000
+---
+
+We're ready to appreciate the blooming talents of our amazing artists once again, in 2018's first fanart contest, featuring spring! Like to draw and want some free osu!supporter? Check this out!
+
+[![](https://assets.ppy.sh/contests/64/header.jpg)](https://osu.ppy.sh/community/contests/64)
+
+The seasons are changing once again, and that means we get a brand new opportunity for a new seasonal-themed fanart contest!
+
+Spring is in the air, so put some spring in your step and hop over to your trusty drawing implement of choice and get to penning **any of the osu! or community mascots in a spring-themed piece**.
```
Comment (TPGPL) [inline news/2018-04-03-spring-2018-fanart-contest.md]: piece -> work of art?

File: news/2018-04-03-spring-2018-fanart-contest.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+The intrepid artists up for the challenge will find they have **16 days** from the date of this post to pull out their best work for the contest.
+
+All valid entries will be placed up for a big community vote pretty much immediately after entry closes for **7 days of agonizing voting**, and the top ten winners will receive **2 months of osu!supporter** for either themselves, a friend, or two friends.
+
+The top ten winners will also be included as 2018's 'seasonal' background pack for spring, and will remain live in the game client for several weeks - conveniently spanning all of the osu!taiko World Cup! Nice.
```
Comment (TPGPL) [inline news/2018-04-03-spring-2018-fanart-contest.md]: Nice. -> Nice!

File: news/2018-04-03-spring-2018-fanart-contest.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+If you're entering, **make sure you read the rules below before you begin.** Some have changed a little, as they tend to do from year to year.
```
Comment (TPGPL) [inline news/2018-04-03-spring-2018-fanart-contest.md]: If you are planning to enter
make sure to read

File: news/2018-04-03-spring-2018-fanart-contest.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+* **Entries MUST meet the resolution requirements for a main-menu piece - IDEALLY 2732x1536, or 1366x768. Entries that don't meet the resolution requirements exactly will be discarded.** We do a little processing on entries before they hit voting stage, and this processing simply does not work if all entries are not of the same resolution. Please don't have your work lost by not paying attention to this!
+* **All entries must be spring themed**. Costume, setting, props - however you do it, it just has to be vaguely spring themed *somehow*. This is a seasonal fanart contest, after all.
+* **Submit only your own work! This means NO TRACING! NO REDRAWING!** Please don't be one of *those* people and submit someone else's work. We want to see your best, not the best of pixiv's hottest artist redrawn or traced.
```
Comment (TPGPL) [inline news/2018-04-03-spring-2018-fanart-contest.md]: and submit > who submit

File: news/2018-04-03-spring-2018-fanart-contest.md (hunk 2 above)
Comment (Ephemeralis) [reply news/2018-04-03-spring-2018-fanart-contest.md]: works better as is

File: news/2018-04-03-spring-2018-fanart-contest.md (hunk 3 above)
Comment (Ephemeralis) [reply news/2018-04-03-spring-2018-fanart-contest.md]: nice is like, conversational ender here, not exclamation

File: news/2018-04-03-spring-2018-fanart-contest.md (hunk 4 above)
Comment (Ephemeralis) [reply news/2018-04-03-spring-2018-fanart-contest.md]: applied

File: news/2018-04-03-spring-2018-fanart-contest.md (hunk 5 above)
Comment (Ephemeralis) [reply news/2018-04-03-spring-2018-fanart-contest.md]: applied

**Pull request merged**